specification/cat/health/CatHealthRequest.ts

```diff
@@ -26,7 +26,6 @@ import { CatRequestBase } from '@cat/_types/CatBase'
  */
 export interface Request extends CatRequestBase {
   query_parameters: {
-    include_timestamp?: boolean
```

Comment:
Leftover from elasticsearch-net, I went through https://github.com/elastic/elasticsearch-net/search?q=RenameQueryStringParams to find all the remaining issues and fixed them all here.